demo/api.md

```diff
@@ -19,6 +19,7 @@ The auro-select element is a wrapper for auro-dropdown and auro-menu to create a
 | [disabled](#disabled)                      | `disabled`                      | `Boolean` |                        | When attribute is present element shows disabled state. |
 | [error](#error)                         | `error`                         | `String`  |                        | When defined, sets persistent validity to `customError` and sets `setCustomValidity` = attribute value. |
 | [noCheckmark](#noCheckmark)                   | `noCheckmark`                   | `Boolean` |                        | When true, checkmark on selected option will no longer be present. |
+| [noMatchWidth](#noMatchWidth)                  | `noMatchWidth`                  | `Boolean` |                        | If set, makes dropdown bib width match the size of the content, rather than the width of the trigger. |

```

Comment:
I think in this case we can use a more semantic appropriate name for select.

In dropdown it makes sense to use `matchWidth` because that is semantically what it does.

It might make more sense to call the attribute in select something like `flexMenuWidth`, `autoMenuWidth` or something to that affect.